webui/react/src/components/Table/TableBatch.tsx

```diff
@@ -12,20 +12,14 @@ interface Action<T> {
 
 interface Props<T> {
   actions?: Action<T>[];
-  ids?: string[];
   onAction?: (action: T) => void;
   onClear?: () => void;
   selectedRowCount?: number;
 }
 
-const defaultProps = {
```

Comment:
to resolve `Support for defaultProps will be removed from function components in a future major release. Use JavaScript default parameters instead.`, removed `defaultProps` and use JS default parameters